<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionPage</class>
 <widget class="QMainWindow" name="VersionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>1091</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="ModListView" name="packageView">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="headerHidden">
         <bool>false</bool>
        </property>
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QLineEdit" name="filterEdit">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="filterLabel">
          <property name="text">
           <string>Filter:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="MCModInfoFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="WideBar" name="toolBar">
   <property name="windowTitle">
    <string>Actions</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionChange_version"/>
   <addaction name="actionMove_up"/>
   <addaction name="actionMove_down"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
   <addaction name="actionCustomize"/>
   <addaction name="actionEdit"/>
   <addaction name="actionRevert"/>
   <addaction name="separator"/>
   <addaction name="actionInstall_Forge"/>
   <addaction name="actionInstall_Fabric"/>
   <addaction name="actionInstall_LiteLoader"/>
   <addaction name="actionInstall_mods"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_to_Minecraft_jar"/>
   <addaction name="actionReplace_Minecraft_jar"/>
   <addaction name="actionAdd_Empty"/>
   <addaction name="separator"/>
   <addaction name="actionMinecraftFolder"/>
   <addaction name="actionLibrariesFolder"/>
   <addaction name="separator"/>
   <addaction name="actionReload"/>
   <addaction name="actionDownload_All"/>
  </widget>
  <action name="actionChange_version">
   <property name="text">
    <string>Change version</string>
   </property>
   <property name="toolTip">
    <string>Change version of the selected package.</string>
   </property>
  </action>
  <action name="actionMove_up">
   <property name="text">
    <string>Move up</string>
   </property>
   <property name="toolTip">
    <string>Make the selected package apply sooner.</string>
   </property>
  </action>
  <action name="actionMove_down">
   <property name="text">
    <string>Move down</string>
   </property>
   <property name="toolTip">
    <string>Make the selected package apply later.</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove selected package from the instance.</string>
   </property>
  </action>
  <action name="actionCustomize">
   <property name="text">
    <string>Customize</string>
   </property>
   <property name="toolTip">
    <string>Customize selected package.</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit selected package.</string>
   </property>
  </action>
  <action name="actionRevert">
   <property name="text">
    <string>Revert</string>
   </property>
   <property name="toolTip">
    <string>Revert the selected package to default.</string>
   </property>
  </action>
  <action name="actionInstall_Forge">
   <property name="text">
    <string>Install Forge</string>
   </property>
   <property name="toolTip">
    <string>Install the Minecraft Forge package.</string>
   </property>
  </action>
  <action name="actionInstall_Fabric">
   <property name="text">
    <string>Install Fabric</string>
   </property>
   <property name="toolTip">
    <string>Install the Fabric Loader package.</string>
   </property>
  </action>
  <action name="actionInstall_LiteLoader">
   <property name="text">
    <string>Install LiteLoader</string>
   </property>
   <property name="toolTip">
    <string>Install the LiteLoader package.</string>
   </property>
  </action>
  <action name="actionInstall_mods">
   <property name="text">
    <string>Install mods</string>
   </property>
   <property name="toolTip">
    <string>Install normal mods.</string>
   </property>
  </action>
  <action name="actionAdd_to_Minecraft_jar">
   <property name="text">
    <string>Add to Minecraft.jar</string>
   </property>
   <property name="toolTip">
    <string>Add a mod into the Minecraft jar file.</string>
   </property>
  </action>
  <action name="actionReplace_Minecraft_jar">
   <property name="text">
    <string>Replace Minecraft.jar</string>
   </property>
  </action>
  <action name="actionAdd_Empty">
   <property name="text">
    <string>Add Empty</string>
   </property>
   <property name="toolTip">
    <string>Add an empty custom package.</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload all packages.</string>
   </property>
  </action>
  <action name="actionDownload_All">
   <property name="text">
    <string>Download All</string>
   </property>
   <property name="toolTip">
    <string>Download the files needed to launch the instance now.</string>
   </property>
  </action>
  <action name="actionMinecraftFolder">
   <property name="text">
    <string>Open .minecraft</string>
   </property>
   <property name="toolTip">
    <string>Open the instance's .minecraft folder.</string>
   </property>
  </action>
  <action name="actionLibrariesFolder">
   <property name="text">
    <string>Open libraries</string>
   </property>
   <property name="toolTip">
    <string>Open the instance's local libraries folder.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModListView</class>
   <extends>QTreeView</extends>
   <header>ui/widgets/ModListView.h</header>
  </customwidget>
  <customwidget>
   <class>MCModInfoFrame</class>
   <extends>QFrame</extends>
   <header>ui/widgets/MCModInfoFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WideBar</class>
   <extends>QToolBar</extends>
   <header>ui/widgets/WideBar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
